orbital-mechanics/Lecture4/Lecture4.ipynb

```diff
@@ -0,0 +1,397 @@
+{
```

Comment:
<p>Over here, I recommend the following:</p><ol><li>Please remove the tags for the equations. The equation numbering will come automatically if you follow point 2 below.</li><li>The equations will need rewriting. You can do this based on the conversation with @mitanshc in the link below. Based on the points made in the below link (especial 2 and 3 in it), please make the same kinds of changes in this Lecture 4 file. Using ChatGPT will significantly speed up your work here. I assume he can also be of some advisory help to you, if needed: <a href="https://github.com/angadhn/SpacecraftDynamics/pull/3#discussion_r1679477576" target="_blank">https://github.com/angadhn/SpacecraftDynamics/pull/3#discussion_r1679477576</a></li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angadhn/SpacecraftDynamics/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>